### PR TITLE
doc: concurrency of gc delete-range

### PR DESCRIPTION
**Pull request by ekexium**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Modify the scope of system variable `tidb_gc_concurrency`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/18117
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -2933,9 +2933,13 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
 - Type: Integer
 - Default value: `-1`
-- Range: `[1, 256]`
+- Range: `{-1} ∪ [1, 256]`
```
Comment (Oreoxmt) [inline system-variables.md:2936]:
```suggestion
- Range: `-1` or `[1, 256]`
```

File: system-variables.md [hunk 2]
```diff
@@ -2933,9 +2933,13 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 - Unit: Threads
-- Specifies the number of threads in the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of GC. A value of `-1` means that TiDB will automatically decide the number of garbage collection threads to use.
+- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) and [Delete Range](/garbage-collection-overview.md#Delete-Ranges) steps of the GC (Garbage Collection) process.
```
Comment (Oreoxmt) [inline system-variables.md:2938]:
```suggestion
- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) and [Delete Range](/garbage-collection-overview.md#Delete-Ranges) steps of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.
```

File: system-variables.md [hunk 3]
```diff
@@ -2933,9 +2933,13 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+- By default, this variable is `-1`, allowing TiDB to automatically determine the appropriate number of threads based on workloads.
+- When this variable is set to a number in the range of `[1, 256]`:
+    - Resolve Locks directly uses the value set for this variable as the number of threads.
+    - Delete Range uses 1/4 of the value set for this variable as the number of threads.
```
Comment (Oreoxmt) [inline system-variables.md:2942]:
```suggestion
    - Delete Range uses one-fourth of the value set for this variable as the number of threads.
```
https://developers.google.com/style/slashes#slashes-with-fractions

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 4]
```diff
@@ -2933,9 +2933,13 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
 - Type: Integer
 - Default value: `-1`
-- Range: `[1, 256]`
+- Range: `-1` or `[1, 256]`
 - Unit: Threads
-- Specifies the number of threads in the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of GC. A value of `-1` means that TiDB will automatically decide the number of garbage collection threads to use.
+- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) and [Delete Range](/garbage-collection-overview.md#Delete-Ranges) steps of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.
```
Comment (qiancai) [inline system-variables.md:2938]:
```suggestion
- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.
- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#Delete-Ranges) step of the GC process.
```

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 5]
```diff
@@ -2933,9 +2933,14 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
 - Type: Integer
 - Default value: `-1`
-- Range: `[1, 256]`
+- Range: `-1` or `[1, 256]`
 - Unit: Threads
-- Specifies the number of threads in the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of GC. A value of `-1` means that TiDB will automatically decide the number of garbage collection threads to use.
+- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.
+- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#Delete-Ranges) step of the GC process.
```
Comment (qiancai) [inline system-variables.md:2939]:
```suggestion
- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#delete-ranges) step of the GC process.
```

File: system-variables.md [hunk 6]
```diff
@@ -2933,9 +2933,14 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
 - Type: Integer
 - Default value: `-1`
-- Range: `[1, 256]`
+- Range: `-1` or `[1, 256]`
 - Unit: Threads
-- Specifies the number of threads in the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of GC. A value of `-1` means that TiDB will automatically decide the number of garbage collection threads to use.
+- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.
+- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#delete-ranges) step of the GC process.
```
Comment (qiancai) [inline system-variables.md:2939]:
```suggestion
- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Ranges](/garbage-collection-overview.md#delete-ranges) step of the GC process.
```

**Pull request merged**